src/Management/management/Api/IndicesApi.php

```diff
@@ -0,0 +1,2933 @@
+            }
+        } elseif (count($formParams) > 0) {
+            if ($multipart) {
+                $multipartContents = [];
+                foreach ($formParams as $formParamName => $formParamValue) {
+                    $multipartContents[] = [
+                        'name' => $formParamName,
+                        'contents' => $formParamValue
+                    ];
+                }
+                // for HTTP post (form)
+                $httpBody = new MultipartStream($multipartContents);
+
+            } elseif ($headers['Content-Type'] === 'application/json') {
+                $httpBody = \GuzzleHttp\json_encode($formParams);
+
+            } else {
+                // for HTTP post (form)
+                $httpBody = \GuzzleHttp\Psr7\build_query($formParams);
+            }
+        }
+
+        // this endpoint requires API key authentication
+        $apiKey = $this->config->getApiKeyWithPrefix('Authorization');
+        if ($apiKey !== null) {
+            $headers['Authorization'] = $apiKey;
+        }
+        // this endpoint requires API key authentication
+        $apiKey = $this->config->getApiKeyWithPrefix('Authorization');
+        if ($apiKey !== null) {
+            $headers['Authorization'] = $apiKey;
+        }
+
+        $defaultHeaders = [];
+        if ($this->config->getUserAgent()) {
+            $defaultHeaders['User-Agent'] = $this->config->getUserAgent();
+        }
+
+        $headers = array_merge(
+            $defaultHeaders,
+            $headerParams,
+            $headers
+        );
+
+        $query = \GuzzleHttp\Psr7\build_query($queryParams);
+        return new Request(
+            'GET',
+            $this->config->getHost() . $resourcePath . ($query ? "?{$query}" : ''),
+            $headers,
+            $httpBody
+        );
+    }
+
+    /**
+     * Operation indexCreate
+     *
+     * Creates an index.
+     *
+     * @param  \Swagger\Client\Model\Index $body body (required)
+     * @param  string $hashid Unique id of a search engine. (required)
+     *
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @throws \InvalidArgumentException
+     * @return \Swagger\Client\Model\Index
+     */
+    public function indexCreate($body, $hashid)
+    {
+        list($response) = $this->indexCreateWithHttpInfo($body, $hashid);
+        return $response;
+    }
+
+    /**
+     * Operation indexCreateWithHttpInfo
+     *
+     * Creates an index.
+     *
+     * @param  \Swagger\Client\Model\Index $body (required)
+     * @param  string $hashid Unique id of a search engine. (required)
+     *
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @throws \InvalidArgumentException
+     * @return array of \Swagger\Client\Model\Index, HTTP status code, HTTP response headers (array of strings)
+     */
+    public function indexCreateWithHttpInfo($body, $hashid)
+    {
+        $returnType = '\Swagger\Client\Model\Index';
```

Comment:
if you expect class name, it's better to use class constant like \Namespace\ClassName::class